src/components/Homepage/NewestArticles.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useEffect, useState } from "react";
+import icon from "../../static/svg/homer.svg";
+import { Thumbnail } from "../Thumbnail/Thumbnail";
+import { Button } from "../Button";
+
+interface IArticle {
+  id: number;
+  heading: string;
+  insight: string;
+  title_photo: string;
```

Comment:
naming should be `photoTitle`

Reply:
this is name that's returned from backend (this is default laravel convention), so, should I leave it like that or add a few lines to code to rename that key?

Reply:
rename it